### PR TITLE
Fix lr key name in case of param groups

### PR DESCRIPTION
**Pull request by rohitgr7**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?   
- [x] Did you write any new necessary tests?  
- [x] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## What does this PR do?
Fix lr key name in case of param groups.

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by awaelchli** [comment]
ping @SkafteNicki for revew :)

**Comment by SkafteNicki** [comment]
LGTM

**Comment by Borda** [comment]
it seems that these lines are not used in any test, may you pls add test for it?
btw, @williamFalcon it LGTM :rabbit:

File: tests/callbacks/test_callbacks.py [hunk 1]
```diff
@@ -326,3 +327,30 @@ class CurrentTestModel(LightTestOptimizersWithMixedSchedulingMixin,
         'Number of learning rates logged does not match number of lr schedulers'
     assert all([k in ['lr-Adam', 'lr-Adam-1'] for k in lr_logger.lrs.keys()]), \
         'Names of learning rates not set correctly'
+
+
+def test_lr_logger_param_groups(tmpdir):
+    """ Test that learning rates are extracted and logged for single lr scheduler"""
+    tutils.reset_seed()
+
+    class CurrentTestModel(LightTestOptimizerWithParamGroups, LightTrainDataloader, TestModelBase):
```
Comment (Borda) [inline tests/callbacks/test_callbacks.py:336]: Pls use new `EvalModelTemplate` instead and define custom `configure_optimizers` if needed
see: https://github.com/PyTorchLightning/pytorch-lightning/blob/master/tests/base/eval_model_optimizers.py

**Comment by williamFalcon** [comment]
@Borda rebase?

**Comment by Borda** [comment]
> @Borda rebase?

@williamFalcon done

**Pull request merged**